lang/id.po

```diff
@@ -0,0 +1,99 @@
+msgid ""
+msgstr ""
+"Language: id\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=(n==1 ? 0 : 1);\n"
+
+# A place for you to credit yourself!
+# Put something like "Latin translation by Jvlivs Gaivs\n" here.
+msgid "TRANSLATION_AUTHOR\n"
+msgstr "Diterjemahkan ke Bahasa oleh Habib Fikri\n"
+
+msgid "FREE_PACKAGES_COUNT\n"
+msgid_plural "FREE_PACKAGES_COUNT\n"
+msgstr[0] "%d paket free (dari total %s)\n"
+msgstr[1] "%d paket free (dari total %s)\n"
+
+msgid "NONFREE_PACKAGES_COUNT\n"
+msgid_plural "NONFREE_PACKAGES_COUNT\n"
+msgstr[0] "%d paket non-free (dari total %s)\n"
+msgstr[1] "%d paket non-free (dari total %s)\n"
+
+msgid "RMS_HAPPY\n"
+msgstr "Hanya paket free - rms akan bangga!\n"
+
+msgid "RMS_DISAPPOINTED\n"
+msgstr "Lebih dari 10%% paket non-free. Apakah Anda benci kebebasan?\n"
+
+msgid "HELP_USAGE\n"
+msgstr "Penggunaan: vrms-rpm [opsi]\n"
+
+msgid "HELP_OPTION_ASCII\n"
+msgstr "    Tampilkan rms ASCII-art ketika tidak ditemukan paket non-free,\n"
+       "    atau ketika terdapat paket non-free lebih dari 10%% dari total.\n"
+
+msgid "HELP_OPTION_COLOUR\n"
+msgstr "    Mengontrol apakah terminal escape sequences dipergunakan\n"
+       "    untuk mewarnai output. Nilai defaultnya 'auto', yang mana menggunakan output berwarna\n"
+       "    ketika menulis di terminal, tetapi tidak berlaku apabila menulis di file atau pipe.\n"
+
+msgid "HELP_OPTION_DESCRIBE\n"
+msgstr "    Ketika mendaftar paket, sertakan ringkasan paket (deskripsi singkat).\n"
+
+msgid "HELP_OPTION_EXPLAIN\n"
+msgstr "    Ketika mendaftar paket, tampilkan lisensinya\n"
+       "    untuk memastikan apakah termasuk klasifikasi free / non-free.\n"
+
+msgid "HELP_OPTION_HELP\n"
+msgstr "    Menampilkan bantuan dan keluar.\n"
+
+msgid "HELP_OPTION_IMAGE\n"
+msgstr "    Seperti --ascii, tetapi menampilkan gambar menggukanak blok karakter Unicode\n"
+       "    dan kode terminal escape mode 256-warna.\n"
```

Comment:
Make the first line fit in the 80-character limit.

Reply:
Same here - 80 characters from the opening " to the closing ", i.e. including the trailing 4-space indent.